docs/man/man1/kubectl-exec.1

```diff
@@ -19,7 +19,7 @@ Execute a command in a container.
 .SH OPTIONS
 .PP
 \fB\-c\fP, \fB\-\-container\fP=""
-    Container name. If omitted, the first container in the pod will be chosen
+    Container name. If omitted, the first app container in the pod will be chosen
```

Comment:
The users' containers. There's no term `app container` in all k8s docs/code, we use `user container` to distinguish from init containers (if that's the reason you make this change here) in the original discuss #1589